src/main/resources/default/templates/biz/jobs/job.html.pasta

```diff
@@ -109,6 +109,35 @@
     </t:editForm>
 
     <script type="text/javascript">
+        sirius.ready(function () {
+            function onChange() {
+                const formData = sirius.getFormData(sirius.querySelector(".edit-form"));
+                sirius.getJSON("/job/params/___job.getName()", formData).then(handleParameterUpdates);
+            }
+
+            function handleParameterUpdates(json) {
+                for (const name in json.params) {
+                    const _elem = sirius.querySelector('.param-' + name);
+                    let param = json.params[name];
+                    if (param.visible) {
+                        _elem.removeAttribute("hidden");
+                    } else {
+                        _elem.setAttribute("hidden", true);
+                    }
+                }
+            }
+
+            document.querySelectorAll('.edit-form input[name]').forEach(function (input) {
```

Comment:
Optional: `_input` instead of `input` as it is a DOM element
Also check the code below.